### PR TITLE
Token authentication, Python samples

### PR DESCRIPTION
**Pull request by bartlomiejwereszczynski**

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from __future__ import unicode_literals
+from ably import AblyRest
+
+client = AblyRest('{{ApiKey}}')
```
Comment (mattheworiordan) [inline server.py:6]: As with the other tutorial, we need to raise an exception if the API Key has not been set with instructions for the user to add the API key

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.idea
```
Comment (mattheworiordan) [inline .gitignore:1]: Not sure this should be part of this repo either

**Comment by bartlomiejwereszczynski** [comment]
updated version is avalable here: https://github.com/ably/tutorials/pull/11

**Review by mattheworiordan — CHANGES_REQUESTED** [review]
I really don't want to review a whole new PR unless there is a good reason to do so.  Why did you not make the amends to this PR instead?

**Pull request closed without merging**